### DISCUSSION TITLE
A new user is not automatically added to the group

### DISCUSSION THREAD

**Discussion by FedorShlein** [requarks/wiki#7247] [Error / Bug Report]
![photo_2024-06-29_13-33-29](https://github.com/requarks/wiki/assets/76495510/1be526f5-d9f3-4f3c-ab31-9c56f0cfd9b6)
Hi everyone! Unfortunately I encountered an error when a new user cannot automatically get into the group after self-registration.
The user registers in the system, but does not get any group, although in the settings it is set that after registration he should get into the guests' group.
**Authentication strategy:** LDAP
**OS:** Ubuntu 22.04/24.04 started the problem on 22.04 after some update, after moving the base to 24.04 the problem remained, so I assume that the problem is in the base or execution of some script.
**Base:** pgsql 14.12/16
**Node:** 16/18

I have enabled LDAP logging, however, there was no output of value to determine the issue

**I attach the error in text:** 
insert into "userGroups" ("groupId", "userId") values ($1, $2), ($3, $4) returning "groupId" - insert or update on table "userGroups" violates foreign key constraint "usergroups_groupid_foreign"

**In systemd the error looks like this:**
warn: LDAP LOGIN ERROR (c2):  insert into "userGroups" ("groupId", "userId") values ($1, $2), ($3, $4) returning "groupId>

**Comment by kulkesk** [comment]
@NGPixel Hi, issue stil persists with version 2.5.311
I ran into the same problem and there is ZERO info about how to fix it on the internet. Please help
